source/opencl_std_ext_inst.inc

```diff
@@ -53,8 +53,8 @@ ExtInst(minmag, 44, List(OperandId, OperandId))
 ExtInst(modf, 45, List(OperandId, OperandId))
 ExtInst(nan, 46, List(OperandId))
 ExtInst(nextafter, 47, List(OperandId, OperandId))
-ExtInst(pow, 48, List(OperandId, OperandId, OperandId))
-ExtInst(pown, 49, List(OperandId))
+ExtInst(pow, 48, List(OperandId, OperandId))
+ExtInst(pown, 49, List(OperandId, OperandId))
```

Comment:
Also, @a2flo: why does this seem to differ from `pow` and `pown` specification [here](https://www.khronos.org/registry/spir-v/specs/1.0/OpenCL.ExtendedInstructionSet.100.html)?